src/databricks/labs/ucx/cli.py

```diff
@@ -502,5 +502,19 @@ def lint_local_code(
     ctx.local_code_linter.lint(prompts, None if path is None else Path(path))
 
 
+@ucx.command
+def create_federated_catalog(w: WorkspaceClient, prompts: Prompts):
+    """(Experimental) Create federated catalog from current workspace Hive Metastore."""
+    ctx = WorkspaceContext(w)
+    ctx.federation.run()
+
+@ucx.command
+def enable_hms_federation(w: WorkspaceClient, prompts: Prompts, ctx: WorkspaceContext | None = None):
+    """(Experimental) Create federated catalog from current workspace Hive Metastore."""
```

Comment:
This command does not create anything it enables the flag in the settings to create the federated catalog

Reply:
Why is this command needed? Can it be merged with the `create_federated_catalog` instead?

Reply:
We need this command because the locations are created before the create_federated_catalog is executed and we want to have the fallback mode enable for the locations when they are created. This command can be independently used for create_federated_catalog in cases where the federated catalog is already created, and UCX needs to be used to create new locations post federation catalog creation.